ui/lib/core/app/helpers/img-path.js

```diff
@@ -0,0 +1 @@
+export { default } from 'core/helpers/img-path';
```

Comment:
This is right for using a helper in an engine, right? I'm noticing there's a loading lag - wondering if I did something incorrectly 🤔

Reply:
That looks right! Shouldn't be a lag loading the actual component itself. Not sure about the image itself which depend on size and location.

Reply:
Yeah - it's just the image (you can see it in the gif that it shows up a little after the page loads). I haven't actively developed with one on the page, so just wanted to confirm that was working as expected 😄